src/main/webapp/web.xml

```diff
@@ -11,4 +11,103 @@
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+
+    <servlet>
+        <servlet-name>home</servlet-name>
+        <servlet-class>mate.controller.IntersectionController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>home</servlet-name>
+        <url-pattern>/home</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>driverCreate</servlet-name>
+        <servlet-class>mate.controller.DriverCreateController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>driverCreate</servlet-name>
+        <url-pattern>/drivers/driverCreate</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>driverList</servlet-name>
+        <servlet-class>mate.controller.DriverListController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>driverList</servlet-name>
+        <url-pattern>/drivers/driverList</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/drivers/all</url-pattern>
```
why the list? It is not quite clear what the list is. better to explicitly say "all"